moped-database/migrations/1650301717000_alter_project_list_view/up.sql

```diff
@@ -0,0 +1,89 @@
+drop view project_list_view;
+
+CREATE VIEW public.project_list_view AS
+with moped_team_members as (
+  select
+    mpp.project_id,
+    string_agg(DISTINCT concat(mu.first_name, ' ', mu.last_name, ':', mpr.project_role_name), ',') AS project_team_members
+   FROM 
+    public.moped_proj_personnel mpp
+        LEFT JOIN public.moped_users mu ON ((mpp.user_id = mu.user_id))
+        LEFT JOIN public.moped_project_roles mpr ON ((mpp.role_id = mpr.project_role_id))
+      where (mpp.status_id = 1)
+      group by mpp.project_id
+)
+  SELECT 
+    mp.project_uuid,
+    mp.project_id,
+    mp.project_name,
+    mp.project_description,
+    mp.project_description_public,
+    mp.ecapris_subproject_id,
+    mp.project_importance,
+    mp.project_order,
+    mp.current_status,
+    mp.timeline_id,
+    mp.current_phase,
+    mp.end_date,
+    mp.project_length,
+    mp.fiscal_year,
+    mp.capitally_funded,
+    mp.project_priority,
+    mp.date_added,
+    mp.added_by,
+    mp.is_retired,
+    mp.milestone_id,
+    mp.task_order,
+    COALESCE(mp.status_id, 0) AS status_id,
+    mp.updated_at,
+    mtm.project_team_members,
+    me.entity_name AS project_sponsor,
+    string_agg(DISTINCT me2.entity_name, ', ') AS project_partner,
+        CASE
+            WHEN ((mp.status_id = 0) OR (mp.status_id IS NULL)) THEN NULL::text
+            WHEN (mp.status_id = 1) THEN mp.current_phase
+            ELSE mp.current_status
+        END AS status_name,
+    string_agg(task_order_filter->>'display_name', ',') as task_order_name,
+    mp.contractor,
+    mp.purchase_order_number,
+    json_agg(mpf.feature) as project_feature,
+    mt.type_name
+   FROM public.moped_project mp
+   left join moped_team_members mtm on (mp.project_id = mtm.project_id)
```

Comment:
I might kick this line in a couple of spaces and use the SQL convention of capital letters for keywords, but there's nothing wrong with this at all from a technical perspective.